flume-hdfs-sink/src/main/java/com/wipro/ats/bdre/flume/sink/hdfs/AbstractHDFSWriter.java

```diff
@@ -156,6 +158,7 @@ public int getFsDesiredReplication() {
    * @throws IllegalAccessException
    * @throws IllegalArgumentException
    */
+  @SuppressWarnings("squid:S1860")
   public int getNumCurrentReplicas()
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Refactor this method to throw at most one checked exception instead of: java.lang.IllegalAccessException, java.lang.reflect.InvocationTargetException [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1160)